versioned_docs/version-1.1.0/advanced/xmtp-source.md

```diff
@@ -0,0 +1,184 @@
+# Alerts for new XMTP messages
+
+This page will walk through an integration of XMTP as a [source](../integration-overview/alerts-in-depth#source) using the **Notifi React Card** and **Admin Portal**. This will alert an user whenever they receive a new message to any XMTP conversation they have at the time of subscribing. 
```

Comment:
What's Notifi React Card?

Reply:
Our easy plug-and-play solution for creating a Notifi subscription UI.

This is the `version-1.1.0` folder, though. All of the actual docs fixes will go in the `docs` folder.